beesdoo_base/wizard/partner.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+from openerp import models, fields, api
+
+class NewEaterWizard(models.TransientModel):
+    """
+        A transient model for the creation of a eater related to a worker.
+    """
+    _name = 'eater.new.wizard'
+
+    def _get_default_partner(self):
+        return self.env.context['active_id']
+
+    first_name = fields.Char('First Name', required=True)
+    last_name = fields.Char('Last Name', required=True)
+    email = fields.Char('Email')
+    
+    partner_id = fields.Many2one('res.partner', default=_get_default_partner)
+    
+    @api.one
+    def create_new_eater(self):
+        client = self.partner_id.sudo()
```

Comment:
In this case the sudo is not needed. In the cas of the beescard, we don't want regular user to be able to create card without the wizard. But in this case the current user should be able to create a partner, it's just a short cut